setup-elastic-index.sh

```diff
@@ -0,0 +1,60 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# OpenCRVS is also distributed under the terms of the Civil Registration
+# & Healthcare Disclaimer located at http://opencrvs.org/license.
+#
+# Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+# graphic logo are (registered/a) trademark(s) of Plan International.
+
+function wait_for_elasticsearch {
+  local -a args=( '-s' '-D-' '-m15' '-w' '%{http_code}' "http://localhost:9200/" )
```

Comment:
This should accept hostname as a var and we need it to be additionally called by the deploy.sh script from Farajaland

Reply:
I have handled it another way. You can check this [PR](You can check https://github.com/opencrvs/opencrvs-farajaland/pull/584)